jobs/v3/howto/auto_complete_sample.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sample
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	talent "google.golang.org/api/jobs/v3"
+)
+
+// [START auto_complete_job_title]
+
+// jobTitleAutoComplete suggests the job titles of the given companyName based on query.
+func jobTitleAutoComplete(service *talent.Service, parent string, companyName string, query string) (*talent.CompleteQueryResponse, error) {
+	complete := service.Projects.Complete(parent).Query(query).LanguageCode("en-US").Type("JOB_TITLE").PageSize(10)
+	if companyName != "" {
+		complete.CompanyName(companyName)
+	}
+	resp, err := complete.Do()
+	if err != nil {
+		log.Fatalf("Failed to auto complete with query %s in company %s, Err: %v", query, companyName, err)
+	}
+	return resp, err
```

Comment:
Please print something in the response (e.g. the list of job titles), rather than returning the response directly.

Should print to an `io.Writer` (given as the first arg) to make testing easier.